rpa-builder/6.7/modules/ROOT/pages/_partials/oauth-authentication-settings.adoc

```diff
@@ -0,0 +1,32 @@
+
+* *Authentication Settings* 
+** *Auth Provider x509 Cert URL*
++
+Authentication provider certificate URL for the Google API credentials. 
+** *Auth URI* 
++
+OAuth2 authentication URI for the Google API credentials.
+** *Client Email* 
++
+Client email of the Google API credentials. 
+** *Client ID* 
++
+Client ID of the Google API credentials. 
+** *Client x509 Cert URL* 
++
+Client certificate URL for the Google API credentials. 
+** *Credential Type*
++
+Type of Google API credentials. For example, `service_account`.
```

Comment:
```suggestion
Type of Google API credentials, for example, `service_account`
```

Reply:
If you make this change, all descriptions are sentence fragments and don't need the period.

Reply:
I think "type" and "credential" should both be either singular or plural.